driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/fsconfig.bpf.c

```diff
@@ -114,7 +114,7 @@ int BPF_PROG(fsconfig_x,
 			auxmap__store_empty_param(auxmap);
 
 			/* Parameter 6: value_charbuf (type: PT_CHARBUF) */
-			auxmap__store_charbuf_param(auxmap, value_pointer, USER);
+			auxmap__store_charbuf_param(auxmap, value_pointer, MAX_PARAM_SIZE, USER);
```

Comment:
Should we use `MAX_PATH` here?

Reply:
For `FSCONFIG_SET_BINARY`, what would be a good limit for `aux` (being specified as syscall arg) in `auxmap__store_bytebuf_param(auxmap, value_pointer, aux, USER);`.

I guess limiting to `MAX_PARAM_SIZE` might be appropriate?

Reply:
> Should we use MAX_PATH here?
Great catch! ty :) 


> For FSCONFIG_SET_BINARY, what would be a good limit for aux (being specified as syscall arg) in auxmap__store_bytebuf_param(auxmap, value_pointer, aux, USER);.

> I guess limiting to MAX_PARAM_SIZE might be appropriate?

Yeah we already limit it to `MAX_PARAM_SIZE` because `MAX_PARAM_SIZE` is exactly `u16 - 1`, so yes we are doing it implicitly :)